pypdf/annotations/_markup_annotations.py

```diff
@@ -165,8 +165,8 @@ def __init__(
                 ),
                 NameObject("/LE"): ArrayObject(
                     [
-                        NameObject(None),
-                        NameObject(None),
+                        NameObject("/None"),
+                        NameObject("/None"),
```

Comment:
This somehow looks odd. Is this really correct?

Reply:
Yes, I was surprised too, but here's the content of PDF specs:

<body>
<p class="p1"><br></p>
<p class="p2"><br></p>

LE | array | (Optional; PDF 1.4) An array of two names specifying the line ending styles that shall be used in drawing the line. The first and second elements of the array shall specify the line ending styles for the endpoints defined, respectively, by the first and second pairs of coordinates, (x1, y1)and (x2, y2), in the L array. Table 176 shows the possible values. Default value: [/None /None].
-- | -- | --
</body>